README.md

```diff
@@ -139,22 +139,19 @@ gitProperties {
 > ```
 
 
-The `.git` directory for the project should be detected automatically, otherwise it can be specified manually using `dotGitDirectory`:
+If the `.git` directory is not located in the same directory as the Gradle project, you will need to specify it using `dotGitDirectory`:
```

Comment:
This documentation looks wrong to me.  The `.git` location always needs to be specified.

Reply:
It used to work for single module projects, maybe something changed recently?

Reply:
`dotGitDirectory` does NOT need to be specified only in this layout:

```
/project-dir
  /.git
 some-project-file.txt
 build.gradle (with "gitProperties { ... }")
```

If the "build.gradle" is in a subdirectory, you DO need to specify `dotGitDirectory`.